src/google/protobuf/io/tokenizer.cc

```diff
@@ -1002,9 +1002,20 @@ bool Tokenizer::ParseInteger(const std::string& text, uint64_t max_value,
 }
 
 double Tokenizer::ParseFloat(const std::string& text) {
+  double result = 0;
+  if (!TryParseFloat(text, &result)) {
+    LOG(DFATAL)
```

Comment:
blech, `LOG(DFATAL)` is the internal spelling.  You need to make this `GOOGLE_LOG(DFATAL)`

Reply:
Weird. I could have sworn I ran tests locally before pushing...

Reply:
done